tests/CMakeLists.txt

```diff
@@ -19,13 +19,16 @@ option(CODE_COVERAGE "Enable generating code coverage with gcov." OFF)
 include(rapids-test)
 rapids_test_init()
 
+# Ensure tests are using the conda env, so they have the correct release/debug compile flags
+rapids_cmake_support_conda_env(conda_env)
```

Comment:
Is there any reason not to also use `MODIFY_PREFIX_PATH` in case we need anything from there?

Reply:
SGTM.